modular_citadel/code/modules/reagents/chemistry/reagents/fermi_reagents.dm

```diff
@@ -0,0 +1,1596 @@
+ //Fermichem!!
+//Fun chems for all the family
+
+/datum/reagent/fermi
+	name = "Fermi" 	//Why did I putthis here?
+	id = "fermi"	//It's meeee
+	taste_description	= "affection and love!"
+	var/ImpureChem 			= "fermiTox"			// What chemical is metabolised with an inpure reaction
+	var/InverseChemVal 		= 0.25					// If the impurity is below 0.5, replace ALL of the chem with InverseChem upon metabolising
+	var/InverseChem 		= "fermiTox" 	// What chem is metabolised when purity is below InverseChemVal, this shouldn't be made, but if it does, well, I guess I'll know about it.
+	var/DoNotSplit			= FALSE				// If impurity is handled within the main chem itself
+	var/OnMobMergeCheck		= FALSE
+	//var/addProc 			= FALSE //When this reagent is added to a new beaker, it does something. Implemented but unused.
+
+//Called when added to a beaker without any of the reagent present. #add_reagent
+/datum/reagent/fermi/proc/FermiNew(holder)
```

Comment:
typecast the holder (specify type like datum/reagents/holder) if it is always that kind of holder.

Reply:
Deleted